pagopa/io-app
# chore: [IOBP-576] Payment manual input A11y and UX fixes

## Short description
Updated A11y and UX for `inputNoticeNumberScreen` and `InputFiscalCodeScreen` under `WalletPayment`

## List of changes proposed in this pull request
those changes apply to both screens:
- removed `disabled` check for the footer CTA
- the same CTA now closes the keyboard, forcing a validity check and UI feedback on the `textInput`
- A11y focus is also brought back to the `textInput` in order to avoid A11y problems.

## How to test
With the `dev-server` active, navigate to wallet>pay>manual input ("digita") and make sure that:
- the `continue` button is always enabled
- when pressing it after typing in the text field, it displays feedback based on the input's validity
- the code `0000000000000000000` (18x0, or any valid TRX code) navigates to the fiscal code input screen
- the same applies for the fiscal code input screen
- voiceOver/talkback correctly moves the a11y focus back to the text input on CTA press
